routes/courses.js

```diff
@@ -5,13 +5,24 @@ const Course = require('../models/course')
 
 /* GET Courses listing. */
 router.get('/', function (req, res, next) {
+
+  const pagenumber = typeof req.query.page === 'undefined' ? 0 : (req.query.page - 1 );
+
   res.setHeader('Content-Type', 'application/json');
-  Course.find()
+  Course.find({},{}, {skip: 10 * pagenumber, limit:10})
```

Comment:
Returns at most 10 courses `limit:10` while skipping some number of courses depending on the page `skip: 10 * pagenumber`